src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -1,10 +1,12 @@
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
 using System.Xml.XPath;
-using Microsoft.OpenApi.Models;
```

Comment:
Automatically sorted usings. Should I revert this change?

Reply:
No, that's fine.